sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentLine.cs

```diff
@@ -39,5 +44,35 @@ private IReadOnlyList<float> Polygon
                 BoundingPolygon = ClientCommon.CovertToListOfPointF(value);
             }
         }
+
+        /// <summary>
+        /// Returns the list of <see cref="DocumentWord"/> that compose this line.
+        /// </summary>
+        /// <returns>The list of <see cref="DocumentWord"/> that compose this line.</returns>
+        public IReadOnlyList<DocumentWord> GetWords()
```

Comment:
Spans are sorted, so we could consider making a binary search to speed up the process. Lines tend to be pretty short, though, so I'm not sure if it's worth the trouble.

Reply:
Should we add a perf test for this scenario especially memory perf testing? You do not have to block GA release for this - but having a follow-up bug in the backlog so we continue to read the trend.

Reply:
Also since spans are sorted and we rely on that service behavior - add a test for it.

Reply:
Created an issue, as suggested: https://github.com/Azure/azure-sdk-for-net/issues/30914

The current is not the smartest, so it doesn't depend on the fact that `Words` is sorted. I do have a test, though.

Reply:
Added the check to our `ValidatePage` helper test method (checked that spans don't overlap and are sorted). It's called by pretty much every document analysis test, so we now have a lot of data showing that the service is following the expected behavior.